lint/lint.go

```diff
@@ -67,8 +67,12 @@ var checkers = map[string]func(c *Context) Checker{
 }
 
 // NewChecker returns checker that implements check of specified name.
-func NewChecker(name string, ctx *Context) Checker {
-	return checkers[name](ctx)
+func NewChecker(name string, ctx *Context) (Checker, bool) {
```

Comment:
I think you have considered returning `nil` here, right?
Could you elaborate why `nil, false` is better than just `nil`? (we still return "dangerous" nil)

Reply:
It just looks easier to understand for me:)
But yes, we could use nil here if u want so

Reply:
You should return `error` instead of `bool` here, like:
```go
return nil, errors.New("checker not found")
```
Otherwise we're leaking that checkers are a map and (nil, false) basically means entry is missing. If there are ever 2 kinds of errors, this `bool` couldn't express that.

On the caller side:
```go
c, err := lint.NewChecker(name, l.ctx)
if err != nil {
	log.Fatalf("%s: %v", err)
}
```

Reply:
I'm okay with 2-value return, but `error` is more usual way to represent failure.
`,ok` is also fine for now, though.
You can decide here on your own.

Reply:
Oh, I see. Thanks.

Reply:
Done